book/src/concepts/build_systems.md

```diff
@@ -28,3 +28,18 @@ When using QML with CXX-Qt [QML modules](https://doc.qt.io/qt-6/qtqml-writing-a-
 This allows for attributes such as `#[qml_element]` to register the QObject with the QML type system without any C++ code.
 
 See [`QmlModule` documentation](https://docs.rs/cxx-qt-build/latest/cxx_qt_build/struct.QmlModule.html) for more details.
+
+## Splitting a project into multiple crates
+
+As your project grows, it can be helpful to organize your code into multiple Rust crates. If your `main` function is
+in Rust ([Cargo is your only build system](../getting-started/4-cargo-executable.md)), simply add the crates as
+dependencies of the top level binary crate in its Cargo.toml file.
+
+If your `main` function is in C++, you can only link one staticlib Rust crate into C++, otherwise linking
```

Comment:
don't know if referring to the CXX section on linking between C++ and Rust is useful too? https://cxx.rs/build/other.html#linking-the-c-and-rust-together

Reply:
I think linking that would be too much information which isn't very relevant for the audience. Though that was helpful to realize `extern crate` is more appropriate than `pub use`